src/QMCDrivers/VMC/VMCBatched.cpp

```diff
@@ -233,7 +213,13 @@ void VMCBatched::runVMCStep(int crowd_id,
   const bool recompute_this_step = (sft.is_recomputing_block && (step + 1) == max_steps);
   // For VMC we don't call this method for warmup steps.
   const bool accumulate_this_step = true;
-  advanceWalkers(sft, crowd, timers, *context_for_steps[crowd_id], recompute_this_step, accumulate_this_step);
+  const bool spin_move            = crowd.get_walker_elecs()[0].get().isSpinor();
```

Comment:
This line can be dangerous if there is no element.
Pass the state via the sft object.
Also use `population_.get_golden_electrons()`

Reply:
done